packet/ipreservations_test.go

```diff
@@ -26,7 +26,7 @@ func TestIPReservationByTags(t *testing.T) {
 	}
```

Comment:
A case with multiple and no tags would be helpful, since this is testing AllTags now. 
```
		{[]string{}, 0},
		{[]string{"b", "c"}, 3},
```

The name of the test function should be updated too (to `AllTags`).

You may want to copy/paste this as is to test `AnyTags`.

Reply:
I'll follow this PR up with a linter/coverage PR: https://github.com/packethost/packet-ccm/compare/master...displague:lint-and-cover?expand=1

Reply:
Yeah, that is a good idea.

Reply:
And doing so caught an error. All fixed now.

Reply:
The `AllTags` test is not testing for cases where multiple tags were expected to match (all of the cases have a single tag).

Neither test is including empty cases (both functions would currently return a 0 index match).

Reply:
Good idea. Got it now. Take another look @displague